### DISCUSSION TITLE
error 1045 in the database

### DISCUSSION THREAD

**Discussion by Malovito** [givanz/VvvebJs#322] [Q&A]
Db error: "Access denied for user 'root'@'localhost' (using password: NO)" Error code: "1045"
error 1045 in the database. Does not allow you to enter the database during installation
Does not allow you to enter the database during installation

**Comment by givanz** [comment]
Hi

What server are you using, do you use xampp or shared hosting with cpanel? 

Can you check if you can connect to mysql with root without password?

Probably root has a password and it doesn't work without password.

You can either create a new user and use it to connect to the database or reset the root password.

Alternatively you can use sqlite that doesn't need configuration instead of mysql.
